AccountService.js

```diff
@@ -5,6 +5,10 @@
 
 import axios from 'axios';
 const headers = {Accept: 'application/ld+json, application/json'};
+/**
+ * this service is used to encapsulate all account
+ * based activity for a project.
+*/
```

Comment:
indent.

Reply:
2 spaces? and that is indent the second line or the first?

Reply:
the asterisks are supposed to line up.

Reply:
- [ ] align asteriks